smithy-aws-protocol-tests/model/restJson1/http-payload.smithy

```diff
@@ -281,6 +281,20 @@ apply HttpPayloadWithStructure @httpResponseTests([
     }
 ])
 
+apply HttpPayloadWithStructure @httpResponseTests([
+    {
+        id: "RestJsonHttpPayloadWithStructureAndEmptyResponseBody",
+        documentation: "Serializes a structure in the payload",
+        protocol: restJson1,
+        code: 200,
+        body: "",
+        bodyMediaType: "application/json",
```

Comment:
Not sure this would be set if there's no payload, or at least, you don't want to enforce it for servers do you?

Reply:
Yeah, probably not, let me get rid of it.